docs/site-example/site.conf

```diff
@@ -190,4 +190,9 @@
   --     show_altitude = false,
   --   },
   -- },
+
+  -- define if the contact field is obligatory
```

Comment:
why? all other optional settings don't have the hint (optional) I think the comments in front make this obvious already